lib/v8debugapi.js

```diff
@@ -385,9 +385,29 @@ module.exports.create = function(logger_, config_, fileStats_) {
 
   function findScripts(scriptPath) {
     var regexp = pathToRegExp(scriptPath);
-    var matches = Object.keys(fileStats).filter(function(script) {
+    var filterScripts = function(script) {
       return regexp.test(script);
-    });
+    };
+    // Use repository relative mapping if present
+    if (config.appPathRelativeToRepository) {
+      var candidate = scriptPath.replace(config.appPathRelativeToRepository,
+        config.workingDirectory);
+      // There should be no ambiguity resolution if project root is provided
```

Comment:
style-guide nit: add '.' at the end of the comment (here an elsewhere on the newly added comments)